app/services/trainees/create_from_dttp.rb

```diff
@@ -209,19 +206,18 @@ def disability
     end
 
     def ethnicity_attributes
-      ethnic_group = Diversities::BACKGROUNDS.select { |_key, values| values.include?(ethnic_background) }&.keys&.first
-
-      if ethnic_background.present? && ethnic_background != Diversities::NOT_PROVIDED
-        diversity_disclosure = Diversities::DIVERSITY_DISCLOSURE_ENUMS[:diversity_disclosed]
-      else
-        diversity_disclosure = Diversities::DIVERSITY_DISCLOSURE_ENUMS[:diversity_not_disclosed]
+      if ethnic_background.present? && ethnic_background == Diversities::NOT_PROVIDED
```

Comment:
In cases where the ethnic_background is "Not known" in DTTP, we need to set the ethnic group to "Not provided". Otherwise the ethnicity section is incomplete.